src/server/controllers/users.controller.js

```diff
@@ -476,8 +512,8 @@ export function editGeneralProfile(req, res, next) {
         S3.putObject(params).promise()
           .then((data, err) => {
             if (data) {
-              let profileUrl = `${S3.endpoint.href}${bucketName}/${imageKey}`;
-              return updateProfile(req, profileUrl);
+              let profileUrl = `${S3_endpoint_href}${bucketName}/${imageKey}`;
```

Comment:
'profileUrl' is never reassigned. Use 'const' instead
'profileUrl' is assigned a value but never used